package.json

```diff
@@ -44,6 +44,7 @@
     "node-gyp": "^9.0.0",
     "nsfw": "^2.2.4",
     "nyc": "^15.0.0",
+    "mkdirp": "^0.5.0",
```

Comment:
Why is this required here?

Reply:
It's used in one of the performance scripts. See:

https://github.com/eclipse-theia/theia/blob/423b14b70b68826ddc8c8fe36e5afa7df5aae641/scripts/performance/extension-impact.js#L21

Reply:
The joy of dealing with implicit dependencies! Thanks for the explanation.